stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -0,0 +1,557 @@
+    _rawValue = unwrapped._rawValue
+  }
+
+  /// Construct ${a_Self} with a given pattern of bits.
+  @_transparent
+  public init?(bitPattern: Int) {
+    if bitPattern == 0 { return nil }
+    _rawValue = Builtin.inttoptr_Word(bitPattern._builtinWordValue)
+  }
+
+  /// Construct ${a_Self} with a given pattern of bits.
+  @_transparent
+  public init?(bitPattern: UInt) {
+    if bitPattern == 0 { return nil }
+    _rawValue = Builtin.inttoptr_Word(bitPattern._builtinWordValue)
+  }
+
+  /// Convert an Unsafe${Mutable}Pointer ${a_Self}.
+  @_transparent
+  public init<T>(_ other: Unsafe${Mutable}Pointer<T>) {
+    _rawValue = other._rawValue
+  }
+
+  /// Convert an Unsafe${Mutable}Pointer ${a_Self}.
+  ///
+  /// Returns nil if `other` is nil.
+  @_transparent
+  public init?<T>(_ other: Unsafe${Mutable}Pointer<T>?) {
+    guard let unwrapped = other else { return nil }
+    _rawValue = unwrapped._rawValue
+  }
+
+%  if not mutable:
+  /// Convert an UnsafeMutableRawPointer ${a_Self}.
+  @_transparent
+  public init(_ other: UnsafeMutableRawPointer) {
+    _rawValue = other._rawValue
+  }
+
+  /// Convert an UnsafeMutableRawPointer ${a_Self}.
+  ///
+  /// Returns nil if `other` is nil.
+  @_transparent
+  public init?(_ other: UnsafeMutableRawPointer?) {
+    guard let unwrapped = other else { return nil }
+    _rawValue = unwrapped._rawValue
+  }
+
+  /// Convert an UnsafeMutablePointer ${a_Self}.
+  @_transparent
+  public init<T>(_ other: UnsafeMutablePointer<T>) {
+    _rawValue = other._rawValue
+  }
+
+  /// Convert an UnsafeMutablePointer ${a_Self}.
+  ///
+  /// Returns nil if `other` is nil.
+  @_transparent
+  public init?<T>(_ other: UnsafeMutablePointer<T>?) {
+    guard let unwrapped = other else { return nil }
+    _rawValue = unwrapped._rawValue
+  }
+%  end # !mutable
+
+%  if mutable:
+  /// Allocate and point at uninitialized memory for `size` bytes with
+  /// `alignedTo` alignment.
+  ///
+  /// - Postcondition: The memory is allocated, but not initialized.
+  public static func allocate(bytes size: Int, alignedTo: Int)
+  -> UnsafeMutableRawPointer {
+    return UnsafeMutableRawPointer(Builtin.allocRaw(
+        size._builtinWordValue, alignedTo._builtinWordValue))
+  }
+%  end # mutable
+
+  /// Deallocate uninitialized memory allocated for `bytes` number of bytes with
+  /// `alignedTo` alignment.
+  ///
+  /// - Precondition: The memory is not initialized.
+  ///
+  /// - Postcondition: The memory has been deallocated.
+  public func deallocate(bytes: Int, alignedTo: Int) {
+    Builtin.deallocRaw(
+      _rawValue, bytes._builtinWordValue, alignedTo._builtinWordValue)
+  }
+  
+  /// Binding the allocated memory to type `T`.
+  /// Returns an Unsafe${Mutable}Pointer<T> to the bound memory at `self`.
+  ///  
+  /// - Precondition: The memory is uninitialized.
+  ///
+  /// - Postcondition: The memory is bound to 'T' starting at `self` continuing
+  ///   through `self` + `capacity` * `strideof(T.self)`
```

Comment:
Doc comments should refer to parameter names, so this line should use `count` instead of `capacity` (or you need to remove `count` from the function signature and keep the default name `capacity`).